sensor/kubernetes/clusterhealth/updater.go

```diff
@@ -223,6 +223,8 @@ func (u *updaterImpl) getLocalScannerInfo() *storage.ScannerHealthInfo {
 		}
 	}
 
+	// TODO/MC: Extend for ScannerV4 case?
```

Comment:
FYI, this is part of scope for ROX-21042 (and the what/how detailed in the ScannerV4 integration design)

Reply:
Good to know, thank you.